content/trust-management/trust-authority.md

```diff
@@ -60,23 +56,163 @@ its list of public signature verification keys by removing the compromised key
+| `blacklist`         | `blacklistedEntities` |      | Entities (sources and other trust authorities) blacklisted by the trust authority.                                                           |
+| `custom`            | `object`              |      | Additional information for custom client implementations.                                                                                    |
+
+<!-- 🔧: Need to do some research on Megolm and Matrix APIs around encryption
+and key management -->
+
+Where:
+
+<!--
+   The definition of `localisedString` here is the same than in source.md.
+   People changing it might want to also change it there (or remove this
+   warning).
+-->
+* `localisedString` is a map associating a [RFC
+  5646](https://tools.ietf.org/html/rfc5646)-compliant language (and variant)
+  identifier to a localisation of the string in the language the identifier
+  refers to.
+* `trustedEntities` is a map associating a Matrix user ID to a JSON object using
+  the following structure:
+
+|    Parameter    |       Type        | Req. |                                                                       Description                                                                            |
+| ----------------| ----------------- | :--: | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `signature`     | `string`          |  x   | Signature generated from the entity's registration event and one of the trust authority's public keys, using one of the algorithm provided under `sig_algo`. |
+| `type`          | `string`          |  x   | Type of the entity. **Must** be one of `source` or `trust_authority`.                                                                                        |
+| `reason`        | `localisedString` |      | Reason to trust the entity.                                                                                                                                  |
+
+
+* `blacklistedEntities` is a map associating a Matrix user ID to a JSON object
+  using the following structure:
+
+|      Parameter      |        Type       | Req. |                                                 Description                                                 |
+| ------------------- | ----------------- | :--: | ----------------------------------------------------------------------------------------------------------- |
+| `reason_code`       | `string`          |  x   | One of the reason codes defined [above](#blacklist-reason-codes).                                           |
+| `after`             | `string`          |  x   | ID of the latest trustworthy event sent by the entity. Empty string if none (e.g. with `B_MISINFORMATION`). |
+| `additional_info`   | `localisedString` |      | More information on the blacklist.                                                                          |
```

Comment:
```suggestion
| `reason`   | `localisedString` |      | Reason given by the TA for why they blacklist this source or other TA.                                                                      |
```
s/additional_info/reason/ also makes it more similar to trustedEntities, which has the same purpose